website/static/js/share/index.js

```diff
@@ -1,3 +1,6 @@
+require('c3/c3.min.css');
```

Comment:
Import the unminified css here. Our build process will minify the CSS in production